projects/kerberos.cmake

```diff
@@ -6,137 +6,139 @@
 xpProOption(kerberos)
 #######################################
 # setup some pathing variables
-# NOTE - Only building for Windows.  Including these libraries on Linux could
-# cause conflicts with OS-installed versions (with care that can be avoided).
-# To enable building on Linux simply remove the outer if(WIN32) and the
-# corresponding endif() (the last one at the end of the file)
-if(WIN32)
-  set(KERBEROS_DOWNLOAD_PATH ${CMAKE_BINARY_DIR}/xpbase/Source/kerberos)
-  if(WIN32)
-    set(KERBEROS_DL_URL http://web.mit.edu/kerberos/dist/kfw/4.0/kfw-4.0.1-src.zip)
-    set(KERBEROS_DL_MD5 6d58ca865beb5b5c145dc4f579753d33)
-    set(KERBEROS_VER 4.0.1)
-  else()
-    set(KERBEROS_DL_URL http://web.mit.edu/kerberos/dist/krb5/1.14/krb5-1.14.tar.gz)
-    set(KERBEROS_DL_MD5 0727968764d0208388b85ad31aafde24)
-    set(KERBEROS_VER 5.1.14)
-  endif()
-  #######################################
-  set(PRO_KERBEROS
-    NAME kerberos
-    WEB "Kerberos" http://web.mit.edu/kerberos/ "Kerberos"
-    LICENSE "open" http://web.mit.edu/kerberos/krb5-1.14/doc/mitK5license.html "MIT Kerberos License information"
-    DESC "Kerberos is a network authentication protocol (only built on Windows)"
-    VER ${KERBEROS_VER}
-    DLURL ${KERBEROS_DL_URL}
-    DLMD5 ${KERBEROS_DL_MD5}
+set(KERBEROS_DOWNLOAD_PATH ${CMAKE_BINARY_DIR}/xpbase/Source/kerberos)
+if(WIN32 OR TRUE) # use this version for all platforms since we're only building on Windows
+  set(KERBEROS_DL_URL http://web.mit.edu/kerberos/dist/kfw/4.0/kfw-4.0.1-src.zip)
+  set(KERBEROS_DL_MD5 6d58ca865beb5b5c145dc4f579753d33)
+  set(KERBEROS_VER 4.0.1)
+elseif(ATTEMPTING_TO_BUILD_ON_LINUX)
+  set(KERBEROS_DL_URL http://web.mit.edu/kerberos/dist/krb5/1.14/krb5-1.14.tar.gz)
+  set(KERBEROS_DL_MD5 0727968764d0208388b85ad31aafde24)
+  set(KERBEROS_VER 5.1.14)
+  # TRICKY: cmake generation of the projects/README.md doesn't support two versions for one project
+  # if two versions needed, there should be two project files -- see externpro 18.01.1 for an example
+  # https://github.com/smanders/externpro/tree/18.01.1/projects (two versions of ffmpeg)
+endif()
```

Comment:
if I didn't add the `OR TRUE` to `if(WIN32)` then the version flip-flops between 4.0.1 and 5.1.14 depending on which OS you were on when the README.md was cmake generated... I tried to add comments to explain what's going on here